nativeio_async_wrapper.js

```diff
@@ -180,6 +192,45 @@ mergeInto(LibraryManager.library, {
     })
   },
 
+  NativeIO_SetLength__deps: ['$NativeIOWrapper','$NativeIOUtils', '$Asyncify'],
+  NativeIO_SetLength: function(fileDescriptor, {{{ defineI64Param('length') }}}) {
+    return Asyncify.handleSleep(function(wakeUp) {
+      {{{ receiveI64ParamAsI32s('length') }}}
+      try {
+        var length = NativeIOUtils.combineI64Params(length_low, length_high);
```

Comment:
Could you add a comment why this is necessary?

Reply:
Added a comment to function.